octobot_trading/exchanges/connectors/ccxt_exchange.py

```diff
@@ -195,17 +344,29 @@ def _get_client_config(self, api_key=None, secret=None, password=None):
         config.update(self.additional_ccxt_config or {})
         return config
 
-    def get_market_status(self, symbol, price_example=None, with_fixer=True):
+    def get_market_status(self, symbol, price_example=None, with_fixer=True, market_status_fixer=None) -> dict:
         try:
-            if with_fixer:
-                return exchanges.ExchangeMarketStatusFixer(self.client.market(symbol), price_example).market_status
-            return self.client.market(symbol)
+            return self.get_fixed_market_status(symbol, price_example=price_example, with_fixer=with_fixer,
+                                                market_status_fixer=market_status_fixer)
         except ccxt.NotSupported:
             raise octobot_trading.errors.NotSupported
         except Exception as e:
             self.logger.error(f"Fail to get market status of {symbol}: {e}")
             return {}
 
+    def get_fixed_market_status(self, symbol, price_example=None, with_fixer=True, market_status_fixer=None) -> dict:
```

Comment:
I don't undersrand why we have two different ways to fix the market status in this method

Reply:
I didn't either at the beginning, that's what it was before, but I moved it from a different place. I'll change that and move it into a dedicated market status parser and remove redundacy